tasks/utils.py

```diff
@@ -151,57 +151,52 @@ def get_git_branch_name():
     return check_output(["git", "rev-parse", "--abbrev-ref", "HEAD"]).strip()
 
 
-def query_version(ctx, git_sha_length=7, match=None):
+def query_version(ctx, git_sha_length=7, prefix=None):
     # The string that's passed in will look something like this: 6.0.0-beta.0-1-g4f19118
     # if the tag is 6.0.0-beta.0, it has been one commit since the tag and that commit hash is g4f19118
-    cmd = "git describe --tags"
-    if match:
-        cmd += " --match \"{}\"".format(match)
+    cmd = "git describe --tags --candidates=50"
+    if prefix and type(prefix) == str:
```

Comment:
If we don't have a prefix (like for the agent), would this not take the latest tag, including a dca tag for the agent ?

Reply:
Mmh yes, you're right. The current logic is broken in both invoke and omnibus then

Reply:
Only matching digits should fix it